app/workers/messaging.php

```diff
@@ -19,7 +19,7 @@
 
 class MessagingV1 extends Worker
 {
-    protected ?SMS $sms = null;
+    protected ?Adapter $sms = null;
```

Comment:
We would love to have a test case for this. We were thinking of updating the Mock adapter to send the secret to the request catcher and then have the e2e test verify. Is that something you'd be able to add?

Reply:
Would love to do it. But I don't have any background with PHP. Yesterday itself I looked over PHP's basic syntax so I cloud contribute to `APPWRITE`. I need some time to fully understand the language.

Reply:
@stnguyen90 
What do you mean by test case?

Reply:
Hey there 👋 I have created tests in a separate PR, it was a pretty complex stuff. I think we can merge your PR, and later merge my PR that improves tests logic.